app/services/weets.js

```diff
@@ -0,0 +1,14 @@
+const axios = require('axios');
+
+const { externalApiError, notFoundError } = require('../errors');
+
+exports.getWeet = () =>
+  axios
+    .get('https://quote-garden.herokuapp.com/api/v3/quotes/random')
```

Comment:
This could be in an environment variable, or maybe create a util with an axios instance to define the bases configurations. What do you think?

Reply:
I agree I will put it in the .env